src/python/dxpy/cli/dataset_utilities.py

```diff
@@ -184,17 +188,18 @@ def extract_dataset(args):
             try:
                 resp_raw = dxpy.DXHTTPRequest(resource=resource_val, data=payload, prepend_srv=False)
                 if 'error' in resp_raw.keys():
-                    if resp_raw['error']['details'] and 'PermissionDenied: Access to the specified database is not allowed' in resp_raw['error']['details']:
-                        print("Insufficient permissions due to the project policy")
+                    if resp_raw['error']['type'] == 'InvalidInput':
+                        print("Insufficient permissions due to the project policy.")
```

Comment:
"project policy" is not too informative. Should it say "project access policy"? Or which policy is it about? The wording should be such that the user ideally can quickly understand by a quick search in our documentation.

Reply:
@keegankelsey, @durae

Reply:
The error messages came originally from PM with security approval on not specifying project policy. We are currently checking 3 different project policies and not specifying which one caused the error in any other cases. 

I'm just trying to explain the origin of those error messages, not that they are correct or we should do it like that. Maybe @keegankelsey can chime in?

Reply:
If it's general on purpose then it can stay as is. Thanks for providing more context @durae!

Reply:
I don't have information as to the rationale for keeping this general. However, if we think of a malicious actor (probably rare use case) then it would be good to minimize any targeted information for follow up on the part of the actor. So, I think keeping it general is fine.